SplunkRumWorkspace/SplunkRum/SplunkRum/Constants.swift

```diff
@@ -0,0 +1,78 @@
+//
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+struct Constants {
+    struct Globals {
+        static let INSTRUMENTATION_NAME = "splunk-ios"
+        static let UNKNOWN_APP_NAME = "unknown-app"
+    }
+    struct SpanNames {
+        static let SPLUNK_RUM_INITIALIZE = "SplunkRum.initialize"
```

Comment:
We write in the docs:

```
   * - ``thread.name``
     - String
     - Name of the thread. ``SplunkRum.initialize`` only applies to the main thread.
```

I take this is unchanged despite the renaming of the initialization function?

Reply:
Yes, this should be unchanged because we are still initializing the SplunkRum object on this thread.